iotdb-cli/src/main/java/org/apache/iotdb/cli/tool/ImportCsv.java

```diff
@@ -283,27 +286,27 @@ private static void loadDataFromCSV(File file, int index) {
       StringBuilder sbd = new StringBuilder();
       ArrayList<Integer> colIndex = entry.getValue();
       sbd.append("insert into " + entry.getKey() + "(timestamp");
-      int skipcount = 0;
+      int skipCount = 0;
       for (int j = 0; j < colIndex.size(); ++j) {
-        if (data[entry.getValue().get(j) + 1].equals("")) {
-          skipcount++;
+        if ("".equals(data[entry.getValue().get(j) + 1])) {
+          skipCount++;
           continue;
         }
         sbd.append(", " + colInfo.get(colIndex.get(j)));
       }
       // define every device null value' number, if the number equal the
       // sensor number, the insert operation stop
-      if (skipcount == entry.getValue().size()) {
+      if (skipCount == entry.getValue().size()) {
         continue;
       }
 
-      // TODO when timestampsStr is empty,
+      // TODO when timestampsStr is empty
       String timestampsStr = data[0];
-      sbd.append(") values(").append(timestampsStr.trim().equals("")
+      sbd.append(") values(").append("".equals(timestampsStr.trim())
           ? "NO TIMESTAMP" : timestampsStr);
 
       for (int j = 0; j < colIndex.size(); ++j) {
-        if (data[entry.getValue().get(j) + 1].equals("")) {
+        if ("".equals(data[entry.getValue().get(j) + 1])) {
```

Comment:
It's the same problem with last comment, it's better to be {data[entry.getValue().get(j) + 1].isEmpty()}